coderd/workspaces_test.go

```diff
@@ -3504,3 +3506,140 @@ func TestWorkspaceUsageTracking(t *testing.T) {
 		require.Greater(t, newWorkspace.LatestBuild.Deadline.Time, workspace.LatestBuild.Deadline.Time)
 	})
 }
+
+func TestNotifications(t *testing.T) {
```

Comment:
Nice tests 👍 
Nit: you could add some more comments to separate `GIVEN`/`WHEN`/`THEN` and explain what you're doing to make them more readable, but they're quite simple and clean already.